.github/ISSUE_TEMPLATE/BugReport.yml

```diff
@@ -0,0 +1,84 @@
+name: Bug Report
+description: Is the SDK not working as expected? Help us improve by submitting a bug report.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Ensure you go through our [troubleshooting](https://docs.datadoghq.com/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/#debugging) page before creating a new issue. 
+        Before getting started, if the problem is urgent or easier to investigate with access to your organization's data please use our [official support channel](https://www.datadoghq.com/support/).
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: Provide a clear and concise description of what the bug is. Include SDK debug logs if possible.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction steps
+      description: |
+        Provide a self-contained piece of code demonstrating the bug. 
+        For a more complex setup consider creating a small app that showcases the problem. 
+        **Note** - Avoid sharing any business logic, credentials or tokens.
+    validations:
+      required: true
+  - type: textarea
+    id: expected_behavior
+    attributes:
+      label: Expected behavior
+      description: Provide a clear and concise description of what you expected the SDK to do.
+    validations:
+      required: false
+  - type: input
+    id: affected_sdk_versions
+    attributes:
+      label: Affected SDK versions
+      description: What are the SDK versions you're seeing this bug in? 
+    validations:
+      required: true
+  - type: input
+    id: last_working_sdk_version
+    attributes:
+      label: Latest working SDK version
+      description: What was the last SDK version that was working as expected?
+    validations:
+      required: true
+  - type: textarea
+    id: device_info
+    attributes:
+      label: Device Information
+      description: |
+        What are the common characteristics of devices you're seeing this bug in. 
+        Specific models, OS versions, network state (wifi / cellular / offline), power state (plugged in / battery), etc.
+    validations:
+      required: false
+  - type: textarea
+    id: other_info
+    attributes:
+      label: Other relevant information
```

Comment:
maybe we can say that Kotlin/Java version is useful to be mentioned here. Gradle version as well.

Logcat logs may be useful also.

Reply:
Initially I included Logcat logs in the `Decribe the bug` section, but we can create a separate one. Maybe that makes more sense. 

I'll add sections for Kotlin/Java versions and Gradle.